packages/app/src/app/pages/Dashboard/Content/routes/Sandboxes/MaxSandboxesBanner.tsx

```diff
@@ -0,0 +1,71 @@
+import { useGetCheckoutURL } from 'app/hooks/useCreateCheckout';
+import { useWorkspaceAuthorization } from 'app/hooks/useWorkspaceAuthorization';
+import { useWorkspaceSubscription } from 'app/hooks/useWorkspaceSubscription';
+import { useAppState } from 'app/overmind';
+import track from '@codesandbox/common/lib/utils/analytics';
+import { dashboard as dashboardUrls } from '@codesandbox/common/lib/utils/url-generator';
+import { Element, MessageStripe } from '@codesandbox/components';
+import { Link } from 'react-router-dom';
+import React from 'react';
+
+export const MaxSandboxesBanner: React.FC = () => {
```

Comment:
nitpick: would call this `RestrictionsBanner` to be compatible with any future message stripe we need to show here

Reply:
@alexnm done here: [086356c](https://github.com/codesandbox/codesandbox-client/pull/7223/commits/086356c8eb43e921aabb350399fda1b122deba08)